frontend/projects/upgrade/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts

```diff
@@ -91,7 +96,9 @@ export class EnrollmentOverTimeComponent implements OnChanges, OnInit, OnDestroy
       this.experiment.conditions.forEach((condition) => {
         this.conditionsFilterOptions.push({ code: condition.conditionCode, id: condition.id });
         this.selectedCondition.push(condition.id);
+        this.conditionsFilterOptions.sort((a, b) => a.code.localeCompare(b.code));
```

Comment:
The sort is done in every loop. This statement can be moved outside of the loop

Reply:
Moved sorting outside the loop